tests/integration-tests/tests/ad_integration/test_ad_integration.py

```diff
@@ -674,25 +544,19 @@ def test_ad_integration(
     config_params = {
         "fsx_supported": fsx_supported,
     }
-    directory_stack_name, nlb_stack_name = directory_factory(
+    directory_stack_name = directory_factory(
         request.config.getoption("directory_stack_name"),
-        request.config.getoption("ldaps_nlb_stack_name"),
```

Comment:
Let's remove this option (--ldaps-nlb-stack-name) from the test runner, which is now unused.